server/service/integration_enterprise_test.go

```diff
@@ -4372,3 +4372,118 @@ func generateNewScriptMultipartRequest(t *testing.T, tmID *uint,
 ) (*bytes.Buffer, map[string]string) {
 	return generateMultipartRequest(t, tmID, "script", fileName, fileContent, token)
 }
+
+func (s *integrationEnterpriseTestSuite) TestAppConfigScripts() {
```

Comment:
Would it make sense to add tests with mixed, non-string values in the json array?

Reply:
good idea, added in https://github.com/fleetdm/fleet/pull/14190/commits/44129a5239e477df9406581de333411ebe5dac6a